src/cpp/shared/game/save.cpp

```diff
@@ -900,9 +914,40 @@ void game::LoadGame(char* filnam, int newGame, int a3) {
       std::map<std::string, mapVariable> mapVariables;
       for (ironfist_save::save_t::mapVariable_const_iterator it = mp->mapVariable().begin();
         it != mp->mapVariable().end(); it++) {
-        mapVariable *mapVar = new mapVariable;
         std::string mapVariableId = it->id().get();
         MapVarType mapVariableType = StringToMapVarType(it->type());
+		if ((mapVariableType == MAPVAR_TYPE_BOOLEAN) && (!mapVariableId.empty())) {
+		  if ((mapVariableId[0] == '_') && ((it->value().get() == "true") || (it->value().get() == "True") || (it->value().get() == "TRUE"))) {
+			if (mapVariableId.length() > 10) {
```

Comment:
Is this redundant with the next check?

Reply:
My logic is to make sure that the format is proper like this: "_AICHASE_X_Y_"